_docs/_latest/develop/website.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Edit & Build Website
+order: 003
+---
+
+#  {{ page.title }}
+
+The instructions below are aimed at developers using Linux, particularly Debian
+or Ubuntu.
+
+---
+
+## High-Level Overview
+
+To build the Forseti Security website locally:
+
+1. [Install Docker](#install-docker)
+2. [Install Ruby & RubyGems](#install-ruby)
+3. [Install Bundler](#install-bundler)
+4. [Install Jekyll](#install-jekyll)
+5. [Fetch, Build, and Run Forseti Web Code](#fetch-code)
+
+## Step by Step Instructions
+
+### Install Docker {#install-docker}
+
+```bash
+# Remove old docker-engine package (if installed)
```

Comment:
We should drop this comment because it's the same as the command.  Instead, a better comment would be why this needs to be removed.